app/components/channel_list/header/header.tsx

```diff
@@ -57,19 +58,54 @@ const getStyles = makeStyleSheetFromTheme((theme: Theme) => ({
 
 const ChannelListHeader = ({displayName, iconPad}: Props) => {
     const theme = useTheme();
+    const isTablet = useIsTablet();
     const serverDisplayName = useServerDisplayName();
     const marginLeft = useSharedValue(iconPad ? 44 : 0);
     const styles = getStyles(theme);
     const animatedStyle = useAnimatedStyle(() => ({
         marginLeft: withTiming(marginLeft.value, {duration: 350}),
     }), []);
 
-    const intl = useIntl();
-
     useEffect(() => {
         marginLeft.value = iconPad ? 44 : 0;
     }, [iconPad]);
 
+    const onPress = useCallback(() => {
+        const renderContent = () => {
+            return (
+                <>
+                    <PlusMenuList
+                        pickerAction='browseChannels'
+                        onPress={() => {
+                            //this is a click
```

Comment:
here you should add the code that you removed for the onPress handler + you should be dismissing the bottom sheet